swarm/src/behaviour.rs

```diff
@@ -230,7 +230,7 @@ pub trait NetworkBehaviour: 'static {
     /// Informs the behaviour about an event generated by the handler dedicated to the peer identified by `peer_id`.
     /// for the behaviour.
     ///
-    /// The `peer_id` is guaranteed to be in a connected state. In other words, `inject_connected`
+    /// The `peer_id` is guaranteed to be in a connected state. In other words, `inject_connection_established`
```

Comment:
Lets change this and other rustdoc appearances to an actual reference with `[``]`.

That will yield a compile-error if we ever hit this again.

Reply:
Thanks for your review. I changed them.